lib/linux-hub/linux_user.rb

```diff
@@ -1,19 +1,36 @@
+require 'fileutils'
+require 'etc'
+
 module LinuxHub
   class LinuxUser
     # The default group is used to keep track of members
     # Members in this group not in the appropriate Github Team are purged
     DEFAULT_GROUP = 'linuxhub'
 
-    def self.users_in_group
-      File.open("/etc/group") do |f|
-        f.each_line do |line|
-          user = line.split(":")
-          if user.first == DEFAULT_GROUP
-            return user[3].chomp.split(',').collect { |u| new(username: u) }
-          end
+    class << self
```

Comment:
My preference is to use the `self.method` as per the [Github styleguide](https://github.com/styleguide/ruby)